### ISSUE TITLE
convox env returns unfriendly error message

### ISSUE THREAD

**Issue by beedub** [convox/rack#659]
I just created an app and tried to run the following:

```bash
$ convox env set TOKEN=my-token
ERROR: BucketRegionError: incorrect region, the bucket is not in 'us-west-2' region
	status code: 301, request id:
```

The unfriendly error seems related to the app still being under creation.

**Comment by awsmsrc** [comment]
this was fixed in #0abbaf24

**Comment by nzoschke** [comment]
Good catch. Closing
